core/primitives/src/version.rs

```diff
@@ -136,12 +138,24 @@ pub enum ProtocolFeature {
 /// Current latest stable version of the protocol.
 /// Some features (e. g. FixStorageUsage) require that there is at least one epoch with exactly
 /// the corresponding version
-#[cfg(not(feature = "nightly_protocol"))]
+#[cfg(all(
```

Comment:
please comment that this should be rolled back to 'NOT nightly'  in January 2022

Reply:
As mentioned in PR description, this would be removed before merge

Reply:
As we are ready for merge I reverted this part